modules/timepoint_list/php/NDB_Menu_timepoint_list.class.inc

```diff
@@ -54,7 +54,11 @@ class NDB_Menu_Timepoint_List extends NDB_Menu
             // create timepoint object
             $timePoint =& TimePoint::singleton($sessionID);
             // check if at least one timepoint belongs to the user's site
-            if ($user->getData('CenterID') == $timePoint->getData('CenterID')) {
+            if (in_array(
```

Comment:
@MounaSafiHarab I would strongly recommend using the strict=true 3rd parameter for in_array()

it works okai in this case but its on the brink of breaking